src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs

```diff
@@ -211,7 +211,8 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
             _coreShell.AssertIsOnMainThread();
             BeforeLoadUnload();
 
-            await _packageManager.LoadPackageAsync(package.Name, package.RepositoryUri?.AbsolutePath.ToRPath());
+            var libPath = await GetLibPath();
+            await _packageManager.LoadPackageAsync(package.Name, libPath);
```

Comment:
No, it should be `await _packageManager.LoadPackageAsync(package.Name, package.LibraryPath.ToRPath())`, otherwise it will fail for system packages.